### PR TITLE
tweaking for new headers

### PR DESCRIPTION
**Pull request by michaelneale**
# Description
cc @tfennelly @sophistifunk @scherler starting to change the styles to match the new header stuff. 

Smokes now passes, as does much of log-karaoke (I need to go to bed now, but will try more tomorro)

### CODE REVIEW

**Comment by sophistifunk** [comment]
😍

File: src/test/js/multibranch/testResults.js [hunk 1]
```diff
@@ -27,9 +27,14 @@ module.exports = {
     },
 
     'Open acitivty page wait for first run to finish': (browser) => {
+        
+
         const blueActivityPage = browser.page.bluePipelineActivity().forJob(jobName);
+        blueActivityPage.click(".branches");
+        blueActivityPage.click(".activity");
```
Comment (tfennelly) [inline src/test/js/multibranch/testResults.js]: Not sure if it's needed (might be covered internally in `click`), but I tend to add an explicit `waitForElementVisible` call before interacting (clicking etc).

File: src/test/resources/test_scripts/freestylePing.sh [hunk 2]
```diff
@@ -1,2 +1,2 @@
 #!/usr/bin/env bash
-ping -c 10 spiegel.de
\ No newline at end of file
+ping -c 10 localhost
```
Comment (tfennelly) [inline src/test/resources/test_scripts/freestylePing.sh]: Why this change? Not sure if it needs to match what's in `initialStage.groovy`.

**Comment by tfennelly** [comment]
@michaelneale @sophistifunk looking good so far. I left a few comments. Is this still a WiP (no label)?

File: src/test/resources/test_scripts/freestylePing.sh (hunk 2 above)
Comment (michaelneale) [inline src/test/resources/test_scripts/freestylePing.sh]: Often the ping fails, and the test fails. could add `||true` at end but why not local host and don't need to hit the network.

File: src/test/js/multibranch/testResults.js (hunk 1 above)
Comment (michaelneale) [inline src/test/js/multibranch/testResults.js]: Yes, it probably should as elsewhere I have seen it fail on clicking things implying that it doesn't wait for it to be visible.

**Comment by michaelneale** [comment]
@tfennelly yes a bit of a WIP but review MUCH appreciated. I have tried to make a few things more reliable as well as adjusting the selectors, mostly minor changes it turns out. The worst one was te selector for the content in "sanityCheck" - Selenium completely barfed with a strange error there... took me a few hours to sort that one out.

**Comment by i386** [comment]
I read the title as "twerking for new headers"

... moving on

**Comment by michaelneale** [comment]
@i386 that is basically what is going on. 

This is me:

**Comment by sophistifunk** [comment]
🐝

File: src/test/js/edgeCases/runDetailsDeepLink.js [hunk 3]
```diff
@@ -30,10 +30,8 @@ module.exports = {
     },
     /** Check Job Blue Ocean Pipeline Activity Page has run */
     'Step 03': function (browser) {
-        const blueActivityPage = browser.page.bluePipelineActivity().forJob(jobName, 'jenkins');
```
Comment (scherler) [inline src/test/js/edgeCases/runDetailsDeepLink.js]: You remove some "assurance code" so the job is in a certain state. Are you sure?

Comment (michaelneale) [inline src/test/js/edgeCases/runDetailsDeepLink.js]: well the code is testing deepLInk - other code tests the activity screen

Comment (michaelneale) [inline src/test/js/edgeCases/runDetailsDeepLink.js]: hrm the comment does imply that, may just add it back in.
